source/includes/_entries.md

```diff
@@ -598,6 +598,21 @@ EntryId     | If the submission was a success, this value will be the EntryId as
 EntryLink   | If the submission was a success, this value will be the URL for an [Entries](#form-entries) request, filtered for this entry
 RedirectUrl | If the form has a [Redirect](http://help.wufoo.com/articles/en_US/SurveyMonkeyArticleType/Form-Settings#confirmation) set up, that URL will be included in the response
 
+### Failed Submissions
+
+>This example shows the server response when the submit entry rate limit is exceeded
+
+```json
+{
+    "Text":"Slow Down",
+    "HTTPCode":429
+}
+```
+
+#### Rate Limit
+
+To prevent malicious bot activity and reduce strain on Wufoo services, entries submitted via the Wufoo v3 API are limited to 50 submissions per Wufoo User in a 5 minute sliding window. Entries which exceed this limit will receive a HTTP 429 response. For more information on this and other Wufoo HTTP Status Codes see [here](#http-status-codes).
+
 >Note this POST request is missing a value for Field105 (a required Text field) and has a text value submitted for Field106 (a Number field)
 
```

Comment:
It seems like this blurb was inserted halfway through another blurb

Reply:
I think the way this works is that the code sections which are shown on the right hand side appear directly before the description sections which are shown on the left.
I added new headers in the failed submissions section to break the failures down between rate limit and bad request. So the other blurb belongs to the bad requests section which appears directly after this one.

I wasn't able to actually test this since the build is very much broken and none of those dependencies work together anymore. So I'm mostly just assuming I'm doing things correctly...